rust/phoenix-sdk/src/sdk_client.rs

```diff
@@ -286,6 +286,7 @@ impl SDKClient {
         let tick_size_in_quote_atoms_per_base_unit =
             header.get_tick_size_in_quote_atoms_per_base_unit().into();
         let num_base_lots_per_base_unit = market.get_base_lots_per_base_unit().into();
+        let raw_base_units_to_base_units = header.raw_base_units_per_base_unit.max(1);
```

Comment:
why `max`? shouldn't this always be set? or is this to support old markets that don't have this field set (and so currently would be read as 0)?

Reply:
can we just redeploy and remove?

Reply:
We can, i don't really want to redeploy the devnet market though. We can probably add something really hacky where in new order we update the header if this field is 0 and then revert the change